styles/LifeAsATorchboxer.module.scss

```diff
@@ -0,0 +1,76 @@
+.container {
+    padding: $grid;
+    display: flex;
+    flex-direction: column;
+
+    @include media-query(medium) {
+        padding: $grid * 2;
+    }
+
+    @include media-query(large) {
+        padding: $grid * 4;
+        flex-direction: row;
+    }
+}
+
+.textContainer {
+    text-align: left;
+    margin-bottom: $grid;
+    order: 1;
+
+    @include media-query(medium) {
+        order: 2;
+    }
+
+    @include media-query(large) {
+        max-width: 635px;
+    }
+}
+
+.title {
+    font-weight: $weight--extrabold;
+    line-height: 60px;
+    @include font-size(xxxxl);
+
+    @include media-query(medium) {
+        line-height: 70px;
+    }
+
+    @include media-query(large) {
+        line-height: 100px;
+    }
+}
+
+.richText {
+    & > p {
+        color: var(--color--dark-indigo);
+    }
+}
+
+.links {
+    @include font-size(m);
+    & a:first-child {
```

Comment:
**Thought:** I think there would usually be a space abover this line, would be good to make another ticket to sort out formatting.